src/electron/menu.ts

```diff
@@ -92,6 +93,53 @@ export function createMenu(store: Store): void {
 						shell.openItem(store.getPreferencesPath());
 					}
 				},
+				{
+					label: 'Export page as PNG',
+					click: () => {
+						const webview = document.getElementById('preview') as WebviewTag;
+
+						// code that gets executed inside the webview to get
+						// the actual size of the preview body
+						const code = `
+							bodyTag = document.querySelector('body');
+							r = {};
+							r.pageHeight = bodyTag.getBoundingClientRect().height;
+							r.pageWidth = bodyTag.getBoundingClientRect().width;
+							r;
+						`;
+
+						webview.executeJavaScript(code, false, webviewSize => {
+							// set the height of the webview tag to the preview body height
+							// This is needed because capturePage can not capture anything that renders
+							// outside the webview area (https://github.com/electron/electron/issues/9845)
+							webview.style.height = webviewSize.pageHeight;
+
+							dialog.showSaveDialog({}, filename => {
```

Comment:
maybe only support files with png extension? (FileFilter or something like that)